_data-prepper/pipelines/configuration/processors/aws-lambda.md

```diff
@@ -20,33 +20,33 @@ You can configure the processor using the following configuration options.
+`aws.sts_role_arn`   | String  | Optional | The Amazon Resource Name (ARN) of the role to assume before invoking the Lambda function. Default is `none`.               
 `max_retries`        | Integer | Optional | The maximum number of retries for failed invocations. Default is `3`.             
-`batch`              | Object  | Optional | The batch settings for the Lambda invocations. Default is `key_name = "events"`. Default threshold is `event_count=100`, `maximum_size="5mb"`, and `event_collect_timeout = 10s`.                            
-`lambda_when`        | String  | Optional | A conditional expression that determines when to invoke the Lambda processor.     
+`batch`              | Object  | Optional | The batch settings for the Lambda invocations. Default is `key_name = "events"`, threshold `event_count=100`, `maximum_size="5mb"`, and `event_collect_timeout = 10s`.                            
+`lambda_when`        | String  | Optional | A conditional expression that determines when to invoke the Lambda processor. Default is `none`.     
 `response_codec`     | Object  | Optional |  A codec configuration for parsing Lambda responses. Default is `json`.
-`tags_on_match_failure` | List | Optional |  A list of tags to add to events when Lambda matching fails or encounters an unexpected error.
-`sdk_timeout`        | Duration| Optional | Configures the SDK's client connection timeout period. Default is `60s`. 
+`tags_on_match_failure` | List | Optional |  A list of tags to add to events when Lambda matching fails or encounters an unexpected error. Default is `[]`.
 `response_events_match` | Boolean | Optional | Specifies how Data Prepper interprets and processes Lambda function responses. Default is `false`.
 `client`            | Object  | Optional | The client configuration.
-`api_call_timeout`   | Duration | Optional | The amount of time that the SDK maintains the API call before timing out, in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601#Durations).
-`base_delay`         | Duration | Optional | The base delay for exponential backoff, in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601#Durations).
-`connection_timeout` | Duration | Optional | The amount of time that the SDK maintains the connection to the client before timing out, in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601#Durations).
-`max_backoff`        | Duration | Optional | The maximum backoff time for exponential backoff, in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601#Durations).
-`max_concurrency`    | Integer  | Optional | The maximum concurrency defined on the client side. 
-`max_retries`        | Integer | Optional | The maximum number of retries before failing.  
+`api_call_timeout`   | Duration | Optional | The amount of time that the SDK maintains the API call before timing out, in [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601#Durations). Default is `60s`.
```

Comment:
It may be better to be more specific. The `api_call_timeout` is actually the total time that the SDK will attempt which includes all retries.